doc/source/examples/permutation_importance_classification_leave.ipynb

```diff
@@ -0,0 +1,933 @@
+{
```

Comment:
<p>If using type hints, it's good practice to also include the return type hint (check other functions in this notebook too please).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/798/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>